### PR TITLE
Remove obsoleted unit tests from root (ZPS-1292)

### PR DESCRIPTION
**Pull request by j053ph4**
- Possible fix for ZPS-1292
- All but one converted for use with ZenPackLib 2
- These files are possibly involved with build failures

### CODE REVIEW

**Comment by dbouchillon** [comment]
changes requested

File: GNUmakefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ##############################################################################
 #
-# Copyright (C) Zenoss, Inc. 2015, all rights reserved.
+# Copyright (C) Zenoss, Inc. 2017, all rights reserved.
```
Comment (dbouchillon) [inline GNUmakefile]: need to include 2015 also.  so 2015-1017

File: GNUmakefile [hunk 2]
```diff
@@ -13,5 +13,8 @@ help:
 	@echo "Please use 'make <target>' where <target> is one of.."
 	@echo "  test    Runs all tests in tests/ directory."
 
+egg:
```
Comment (dbouchillon) [inline GNUmakefile]: ok, there actually needs to be more to egg building.  i think in setup.py we'd need to make some changes.  just remove this part for now.  keep the test section though

**Comment by dbouchillon** [comment]
approved

**Comment by j053ph4** [comment]
thanks

**Pull request merged**